node_modules/gh-rest/lib/rest/orgunit.js

```diff
@@ -293,4 +293,25 @@ module.exports = function(client) {
         var url = '/api/orgunit/' + client.util.encodeURIComponent(id) + '/export';
         client._request(url, 'GET', {'format': format}, null, callback);
     };
+
+    /**
+     * Import organisational units, event series and events under an organisational unit
+     *
+     * @param  {Number}         id                              The id of the organisational unit into which to import the data
+     * @param  {Object}         importData                      The organisational units, event series and events to import
+     * @param  {Boolean}        deleteMissing                   Whether to delete items from the database that aren't in the import data
+     * @param  {Function}       callback                        Standard callback function
+     * @param  {Object}         callback.err                    An error that occurred, if any
+     * @param  {Object}         callback.body                   The JSON response from the REST API
+     * @param  {Response}       callback.response               The response object as returned by requestjs
+     * @see gh-orgunit/lib/rest.js for more information
+     */
+    client.orgUnit.importOrgUnit = function(id, importData, deleteMissing, callback) {
+        var url = '/api/orgunit/' + client.util.encodeURIComponent(id) + '/import';
+        var data = {
+            'data': JSON.stringify(importData),
+            'deleteMissing': deleteMissing
+        };
```

Comment:
We could also consider accepting `Content-Type: application/json` with this data which might be more straightforward and web-like. But I don't think this is a big issue.